packages/editor-ui/src/views/helpers.ts

```diff
@@ -7,6 +7,11 @@ interface ICorners {
 	maxY: number;
 }
 
+interface IZoomConfig {
+	scale: number;
+	offset: XYPositon;
+}
+
```

Comment:
should we maybe move this to `"@/Interface"` as well? same for the `ICorners`?

Reply:
we should only do this for interfaces that we are reusing or might reuse in the future. I can see this being used so will move it, but not `ICorners`... Let me know if you think otherwise.